### PR TITLE
Fixed a spelling mistake

### PR DESCRIPTION
**Pull request by mage2pratik**
## This PR is a:
- [ ] Bug fix or improvement
 

## Summary
 
Fixed a spelling mistake.
 

## Additional information
N/A

### CODE REVIEW

**Comment by shrielenee** [comment]
Hey @mage2pratik - thanks for submitting contributions to DevDocs! I notice that you created quite a few PRs recently, with one change each. In order to best facilitate the merging of these changes, we prefer small changes like these to be batched into one PR.

If you could add the changes you made in the following PRs to this PR, and resubmit it, I'll take a look!
- https://github.com/magento/devdocs/pull/2484
- https://github.com/magento/devdocs/pull/2485
- https://github.com/magento/devdocs/pull/2486

I'll be closing the above-mentioned PRs in anticipation of seeing those changes here. Thanks a ton!

**Comment by mage2pratik** [comment]
Hi @shrielenee - As you've described in the previous comment, I've merged all closed PRs into this PR. Kindly have a look and verify it. Happy to help :)

File: guides/v2.2/frontend-dev-guide/css-topics/css-preprocess.md [hunk 1]
```diff
@@ -78,8 +78,6 @@ To set the compilation mode, do the following:
 <li>In the <b>Store View</b> drop-down field, select <b>Default Config</b>.</li>
 <li>Under <b>Front-end development workflow</b>, in the <b>Workflow type</b> field, select the compilation mode.</li>
 <li>To save the settings, click <b>Save Config</b>.</li>
-
-<li>Make sure that the same compilation mode is set for each configuration scope. That is, check the <b>Front-end development workflow</b> option having switched the <b>Store View</b> drop-down field to the {% glossarytooltip a3c8f20f-b067-414e-9781-06378c193155 %}website{% endglossarytooltip %} scope first, and then to the {% glossarytooltip ca5a9ff1-8182-4fc4-a34b-9b3f831dbf3f %}store view{% endglossarytooltip %}. Change the option to match the default config if it is different.</li>
```
Comment (shrielenee) [inline guides/v2.2/frontend-dev-guide/css-topics/css-preprocess.md]: Why was this section removed?

**Comment by shrielenee** [comment]
Also, when pushing up a PR, make sure you choose `master` as the target branch instead of `develop`. We changed it recently!

**Pull request closed without merging**